packages/common/random-access-storage/src/testing/storage.blueprint-test.ts

```diff
@@ -246,6 +247,47 @@ export const storageTests = (testGroupName: StorageType, createStorage: () => St
       }
     });
 
+    test('reset', async () => {
+      if (
+        testGroupName === StorageType.RAM || // RAM storage does not persist data.
+        testGroupName === StorageType.IDB // IDB storage is blocked by opened connection, and there is no handle to close it.
```

Comment:
it's not possible to reset idb without initializing?

Reply:
It is only possible to reset IDB before initialization because the library we are using for IDBStorage doesn't properly dispose connection to IDB and that is blocking reset after initialization